code/game/machinery/machinery.dm

```diff
@@ -539,13 +539,13 @@ Class Procs:
 
 	if(check_records || check_arrest)
 		var/perpname = perp.get_visible_name(TRUE)
-
 		var/datum/data/record/R = find_security_record("name", perpname)
-		if(check_records && !R)
-			threatcount += 4

```

Comment:
Nearly perfect. We're missing this threat. When beepsky checks records and the person has no record

Reply:
Dang, now I just missed it.